src/transformers/models/whisper/generation_whisper.py

```diff
@@ -759,6 +759,8 @@ def generate_with_fallback(
         do_condition_on_prev_tokens,
         kwargs,
     ):
+        kwargs = dict(kwargs)
```

Comment:
Why do you use `dict(...)` here and below? Is it to copy ? If yes, shouldn't we use `copy.deepcopy` instead ?

Reply:
Yes, it's just to make a copy. My thinking here was that a shallow copy (using `dict()` or `copy.copy()`) has the same effect as using the `**kwargs` syntax.

Reply:
`copy.deepcopy` should make the trick then right ? I'm just afraid that using `dict` might no be self-explanatory

Reply:
I still don't think we want to make a _deep_ copy (what if `kwargs` contains `assistant_model`, for example?). So I changed the `dict` do `copy.copy`, which is equivalent and more readable.